projects/ngrx.io/content/guide/component-store/write.md

```diff
@@ -89,3 +90,42 @@ export class MoviesPageComponent {
   }
 }
 </code-example>
+
+## `patchState` method
```

Comment:
@markostanimirovic 
Thanks for adding this! 👍

Could you please also note here that the state has to be initialized before any of `patchState` calls, otherwise it will throw "not initialized" error?

Reply:
@alex-okrushko
added 🙂